packages/templates/wasm/golang/build.py

```diff
@@ -0,0 +1,106 @@
+import os
+import subprocess
+import re
+
+ACCEPTED_IMPORTS = {'wrap', 'env'}
+
+def build_wam_module():
```

Comment:
this file shouldn't exist, all of this should be done through the CLI's build image